pygmt/tests/test_ternary.py

```diff
@@ -0,0 +1,22 @@
+"""
+Tests ternary.
+"""
+import pytest
+from pygmt import Figure
+
+
+@pytest.mark.mpl_image_compare
+def test_ternary():
+    """
+    Test plotting a ternary chart.
+    """
+    fig = Figure()
+    fig.ternary(
+        table="@ternary.txt",
+        region="0/100/0/100/0/100",
+        cmap="red,orange,yellow,green,blue,violet",
+        projection="6i",
+        frame=["bafg+lAir", "cafg+lLimestone", "aafg+lWater"],
+        style="c0.1c",
```

Comment:
Set an outline pen so that the white dots still show up.
```suggestion
        style="c0.1c",
        pen="thinnest",
```